sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ConsistencyPolicy.java

```diff
@@ -129,4 +135,10 @@ public ConsistencyPolicy setMaxStalenessInterval(Duration maxStalenessInterval)
         super.set(Constants.Properties.MAX_STALENESS_INTERVAL_IN_SECONDS, maxStalenessInterval.getSeconds());
         return this;
     }
+
+    /**
+     * Assumption: all consistency mutations are through setDefaultConsistencyLevel only.
+     * NOTE: If the underlying ObjectNode is mutated cache might be stale
+     */
+    private ConsistencyLevel cachedConsistencyLevel = null;
```

Comment:
in the other places we don't call this "cached" to be consistency please rename to `consistencyLevel`
please see `DocumentCollection` as example.